src/components/ListItemCard.tsx

```diff
@@ -1,16 +1,23 @@
+import Image from "next/image";
+import clsx from "clsx";
+import { FC } from "react";
+import { Button } from "@/components/boilerplate/Button";
 
-'use client';
-import {Button} from '@/components/boilerplate/Button';
-import Image from 'next/image'
-import { ItemAddedPreviewModal } from './ItemAddedPreviewModal';
+interface Props {
+  className?: string;
+  onAddToCart: () => void;
+}
 
-export default function ListItemCard({ className }: any) {
+const ListItemCard: FC<Props> = ({ className, onAddToCart }: Props) => {
   return (
-    <div className={
-      `border border-slate-160
-      flex p-4 bg-white
-      ${className}`
-    }>
+    <div
+      className={clsx(
```

Comment:
`clsx` was added with the scope of grouping the styles, this can be cleaned up even more in the future, but it enables us to identify groups of rules
```
clsx(
        "p-4 bg-white border border-slate-160 gap-4 flex", //comon rules related to styling
        "flex-col items-center", // base layout 
        "lg:flex-row lg:items-start", //specific layout
        className // extra classes
)

```